ports/vc/portfile.cmake

```
@@ -1,24 +1,25 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO  VcDevel/Vc
-    REF 1.4.2
-    SHA512 9a929cd48bdf6b8e94765bd649e7ec42b10dea28e36eff288223d72cffa5f4fc8693e942aa3f780b42d8a0c1824fcabff22ec0622aa8ea5232c9123858b8bbbf
+    REF 1.4.3
+    SHA512 7c0c4ccf8c7c4585334482135f2daf1a5bc088114b880093893583bdcea1fbfcec02485da6059304c510c8b1bb1b768ef04fd7ac8ccb21b9ebbad5d0d5babaef
     HEAD_REF 1.4
-    PATCHES
-        correct_cmake_config_path.patch
-        dont-use-uninit-var.patch
+    PATCHES 
+       correct_cmake_config_path.patch
+       Fix-internal-func-export.patch
```

```suggestion
       Fix-internal-func-export.patch #remove it in next version
```
There needs to be a note that this will remove it in the next release